GDJS/Runtime/inputmanager.ts

```diff
@@ -126,7 +126,7 @@ namespace gdjs {
      * Return true if the key corresponding to the location-aware keyCode was released during the last frame.
      * @param locationAwareKeyCode The location-aware key code to be tested.
      */
-    wasKeyReleased(locationAwareKeyCode: number) {
+    wasKeyReleased(locationAwareKeyCode: string | number) {
```

Comment:
Is this change necessary? locationAwareKeyCode should always be numbers, never strings.

Reply:
No it's not. I added it when arthuro helped me working on the test of this new feature but it's now useless you can remove it if  you want :D